src/app/models/sap_model.php

```diff
@@ -284,4 +284,88 @@ public function submit_review($submission_id, $approved) {
             false
         );
     }
+
+    public function get_users_list() {
+        $stmt = $this->DB->sap->query('SELECT * FROM `sap_ambassadors` WHERE `is_removed` = 0');
+        return $stmt->fetch_all(MYSQLI_ASSOC);
+    }
+
+    public function get_user($id, $activated=true) {
+        $stmt = $this->db_lib->prepared_execute(
+            $this->DB->sap,
+            'SELECT * FROM `sap_ambassadors` WHERE `id`=? AND `has_activated`=?',
+            'ii',
+            [$id, $activated],
+            true
+        );
+        return $stmt->get_result()->fetch_assoc();
+    }
+
+    public function approve_user($id, $password_hash) {
+        return $this->db_lib->prepared_execute(
+            $this->DB->sap,
+            'UPDATE `sap_ambassadors` SET `hash_for_ceating_password`=? WHERE `id`=?',
+            'si',
+            [$password_hash, $id],
+            false
+        );
+    }
+
+    public function remove_user($id) {
+        $this->db_lib->prepared_execute(
```

Comment:
Oops. Not just nice to have, required! I'm using the return value in the controller. :P Fixing.